tests/deployment.ts

```diff
@@ -420,9 +432,11 @@ export async function defaultTestConfig(
   ]);
 
   // Send weth to each exchange.
-  await exchanges.map((exchange) => {
-    return weth.transfer(exchange, utils.parseEther('100'));
-  });
+  await Promise.all(
```

Comment:
This is a fix for a bug that I added in a previous commit accidentally. This did not manifest itself in a test failure because it's probably a rare race condition but I stumbled over this and thought fixing it here doesn't hurt.